tests/MiniProfiler.Tests/App.config

```diff
@@ -34,4 +34,7 @@
       <add name="SQLite Data Provider" invariant="System.Data.SQLite" description=".NET Framework Data Provider for SQLite" type="System.Data.SQLite.SQLiteFactory, System.Data.SQLite" />
     </DbProviderFactories>
   </system.data>
+  <appSettings>
+    <add key="xunit.parallelizeTestCollections" value="false"/>
```

Comment:
The proper fix here is to categorize the tests, which is already happening in some cases. Disabling parallelization just hides actual issues.